src/content/en/2020/http2.md

```diff
@@ -303,94 +370,256 @@ As discussed earlier, HTTP/1.1 only allowed a single request at a time over a TC
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>{{ figure_link(caption="Distribtion of pushed assets on desktop.", sheets_gid="698874709", sql_file="number_of_h2_and_h3_pushed_resources_and_avg_bytes.sql") }}</figcaption>
 </figure>
 
-<figure markdown>
-| Percentile |  HTTP/2 | Size (KB)  | gQUIC | Size (KB) |
-|------------|--------------------|----------------------|------------------------|----------------------|
-| 10         | 1                  | 15.48                | 1                      | 0.06                 |
-| 25         | 1                  | 36.34                | 1                      | 0.06                 |
-| 50         | 3                  | 183.83               | 2                      | 24.06                |
-| 75         | 10                 | 225.41               | 5                      | 204.65               |
-| 90         | 12                 | 351.05               | 18                     | 453.57               |
-
-<figcaption>{{ figure_link(caption="Distribtion of pushed assets on mobile.", sheets_gid="698874709", sql_file="number_of_h2_and_h3_pushed_resources_and_avg_bytes.sql") }}</figcaption>
+<figure>
+  <table>
+    <thead>
+      <tr>
+        <th>Percentile</th>
+        <th>HTTP/2</th>
+        <th>Size (KB)</th>
+        <th>gQUIC</th>
+        <th>Size (KB)</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td class="numeric">10</td>
+        <td class="numeric">1</td>
+        <td class="numeric">15.48</td>
+        <td class="numeric">1</td>
+        <td class="numeric">0.06</td>
+      </tr>
+      <tr>
+        <td class="numeric">25</td>
+        <td class="numeric">1</td>
+        <td class="numeric">36.34</td>
+        <td class="numeric">1</td>
+        <td class="numeric">0.06</td>
+      </tr>
+      <tr>
+        <td class="numeric">50</td>
+        <td class="numeric">3</td>
+        <td class="numeric">183.83</td>
+        <td class="numeric">2</td>
+        <td class="numeric">24.06</td>
+      </tr>
+      <tr>
+        <td class="numeric">75</td>
+        <td class="numeric">10</td>
+        <td class="numeric">225.41</td>
+        <td class="numeric">5</td>
+        <td class="numeric">204.65</td>
+      </tr>
+      <tr>
+        <td class="numeric">90</td>
+        <td class="numeric">12</td>
+        <td class="numeric">351.05</td>
+        <td class="numeric">18</td>
+        <td class="numeric">453.57</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>{{ figure_link(caption="Distribtion of pushed assets on mobile.", sheets_gid="698874709", sql_file="number_of_h2_and_h3_pushed_resources_and_avg_bytes.sql") }}</figcaption>
 </figure>
 
 
 Looking at the frequency of push by content type in Figure 22.20, we see 89% of pages push scripts and 68% push CSS. This makes sense, as these can be small files typically on the critical path to render a page.
 
+{# TODO(analysts): Should this chart differentiate desktop and mobile? #}
```

Comment:
@bazzadp

Reply:
Think it should but struggling to see how @dotjs created this chart to be honest so will let him decide and then fix and I can retake backup image then.

Reply:
Added desktop @bazzadp Could you update the image please.

Reply:
Done.